emuu-client/src/App.css

```diff
@@ -1,5 +1,5 @@
 .App {
-background: black;
+background: #0f0f0f;
```

Comment:
Try making the color a bit lighter, searched it up and it still does look very close to black, but may look different in the application, will have to make a team decision on what looks best.